internal/postprocessor/main.go

```diff
@@ -336,6 +339,26 @@ func (c *config) getDirs() []string {
 	return dirs
 }
 
+func (c *config) MoveSnippets() error {
+	dirs := c.getDirs()
+	for _, dir := range dirs {
+
+		snpDirs, err := filepath.Glob(filepath.Join(dir, "api*", "internal"))
```

Comment:
I would use `apiv` to reduce the chance of false positives. Did you test this locally to see if this would only trigger as expected?

Reply:
Done, thanks!

Reply:
See Pr desc for local test confirmation.